core/src/utilities/helpers/routing-helpers.js

```diff
@@ -54,7 +48,7 @@ class RoutingHelpersClass {
   parseParams(paramsString) {
```

Comment:
Can you also add a test case for this scenario on: **core/test/utilities/helpers/routing-helpers.spec.js**
for the parseParams function.